tests/Stripe/StripeObjectTest.php

```diff
@@ -127,8 +127,21 @@ public function testToArrayRecursive()
 
     public function testNonexistentProperty()
     {
-        $s = new StripeObject();
-        $this->assertNull($s->nonexistent);
+        $capture = \tmpfile();
+        $origErrorLog = \ini_set('error_log', \stream_get_meta_data($capture)['uri']);
+
+        try {
+            $s = new StripeObject();
+            $this->assertNull($s->nonexistent);
+
+            $this->assertRegExp(
+                "/Stripe Notice: Undefined property of Stripe\\\\StripeObject instance: nonexistent/",
+                \stream_get_contents($capture)
+            );
+        } finally {
+            \ini_set('error_log', $origErrorLog);
+            \fclose($capture);
+        }
```

Comment:
While running the test suite, I noticed that this test was printing "Stripe Notice: Undefined property of Stripe\StripeObject instance: nonexistent" to stdout. I used the same technique as `DefaultLoggerTest` to capture the output. This has two upsides: the test suite's output is clean, and we actually test the behavior that accessing a non-existent property outputs a notice to the logger.